armadillo/src/main/java/org/molgenis/armadillo/storage/ArmadilloStorageService.java

```diff
@@ -84,6 +82,46 @@ public void copyObject(String project, String newObject, String oldObject) {
     storageService.save(inputStream, SHARED_PREFIX + project, newObject, APPLICATION_OCTET_STREAM);
   }
 
+  @PreAuthorize("hasRole('ROLE_SU')")
+  public void createLinkedObject(
+      String sourceProject,
+      String sourceObject,
+      String linkName,
+      String linkProject,
+      String variables)
+      throws IOException {
+    throwIfUnknown(sourceProject, sourceObject + PARQUET);
+    throwIfUnknown(linkProject);
+    throwIfDuplicate(linkProject, linkName + LINK_FILE);
```

Comment:
They already existed and are used throughout armadillo. I don't really feel comfortable changing them since it impacts more than just this. I also think they are quite self-explanatory.